internal/support/notifications/rest_notification.go

```diff
@@ -385,16 +484,9 @@ func notificationsByLabelsHandler(w http.ResponseWriter, r *http.Request) {
 
 	labels := splitVars(vars["labels"])
 
-	n, err := dbClient.GetNotificationsByLabels(labels, limitNum)
+	n, err := getNotificationsByLabels(labels, limitNum)
 	if err != nil {
-		if err == db.ErrNotFound {
-			http.Error(w, "Notification not found", http.StatusNotFound)
-		} else {
-			http.Error(w, err.Error(), http.StatusInternalServerError)
-		}
-		LoggingClient.Error(err.Error())
-		w.Header().Set("Content-Type", applicationJson)
```

Comment:
Please do not remove the header set.

Reply:
I think it is not necessary in this case. http.Error handles the Content-Type